scripts/tidy.sh

```diff
@@ -2,7 +2,25 @@
 
 set -euo pipefail
 
-for f in $(git ls-files | grep go.mod)
+ROOT_DIR=$(git rev-parse --show-toplevel)
+SDK_DIR="$ROOT_DIR/sdk"
+PKG_DIR="$ROOT_DIR/pkg"
+
+for f in $(git ls-files | grep 'go\.mod')
```

Comment:
git ls-files can filter down to go.mod files.

```suggestion
for f in $(git ls-files '**/go.mod')
```

Reply:
Ah, thank you, I didn't know it understood that glob syntax.